src/pure/parsers/helper.ts

```diff
@@ -29,6 +29,7 @@ const commonPlugins: ParserPlugin[] = [
   'optionalChaining',
   'partialApplication',
   'throwExpressions',
+  'explicitResourceManagement',
```

Comment:
Can we add a comment specifying that this relates to "using" keyword?

Reply:
Sure! Done! 👍